playground/backend/internal/preparers/java_preparers.go

```diff
@@ -103,10 +138,38 @@ func GetJavaPreparers(builder *PreparersBuilder, isUnitTest bool, isKata bool) {
 	if isKata {
 		builder.JavaPreparers().
 			WithPublicClassRemover().
-			WithPackageRemover()
+			WithPackageRemover().
+			WithGraphHandler()
 	}
 }
 
+// findPipelineObjectName finds name of pipeline in JAVA code when pipeline creates
+func findPipelineObjectName(filepath string) (string, error) {
+	reg := regexp.MustCompile(pipelineNamePattern)
+	b, err := ioutil.ReadFile(filepath)
+	if err != nil {
+		return "", err
+	}
+	matches := reg.FindStringSubmatch(string(b))
+	if len(matches) > 0 {
+		return matches[1], nil
+	} else {
+		return "", nil
+	}
+
+}
+
+func findImports(filepath string) ([]string, error) {
+	reg := regexp.MustCompile(findImportsPattern)
+	b, err := ioutil.ReadFile(filepath)
+	if err != nil {
+		return nil, err
+	}
+	imports := reg.FindAllString(string(b), -1)
+
+	return imports, nil
+}
```

Comment:
Please, delete this function and tests for it as we discussed.

Reply:
Done